README.md

```diff
@@ -9,13 +9,34 @@ The ohsome API provides various endpoints for [data aggregation](https://api.ohs
 
 ## Installation
 
-*ohsome-py* requires Python >= 3.6. The easiest way to install *ohsome-py* is using pip:
+*ohsome-py* requires
+
+* Python >= 3.6
+* geopandas >= 0.9.0
```

Comment:
wouldn't goepandas 0.8.0 be sufficient? thats what conda currently has (I#m not advocating for conda, all of these env-managers are cumbersome, just trying to make it the most simple for all users)

Reply:
Yes, I thought the same, but for some reason some test fail with geopandas 0.8.0. So I'd leave it like that and test it again when I have time.

Reply:
Yes, I thought the same, but for some reason some test fail with geopandas 0.8.0. So I'd leave it like that and test it again when I have time.